sdk_experimental/lib/opentelemetry/sdk/experimental.rb

```diff
@@ -14,3 +14,4 @@ module Experimental
 end
 
 require 'opentelemetry/sdk/experimental/samplers_patch'
+require 'opentelemetry/sdk/trace/propagation/trace_context/response_text_map_propagator'
```

Comment:
I’m going to be a bastard and ask you to jump through hoops like this: https://github.com/open-telemetry/opentelemetry-ruby/blob/32f4a7a10ecce597a574a675fa797e29561c473d/sdk_experimental/lib/opentelemetry/sdk/experimental/samplers_patch.rb#L7-L13

The rationale is that some point we _might_ want to move the functionality to the SDK-proper, and we don’t want to break folks using the `-sdk-experimental` package for other things.